bin/get-schemas-for-redoc

```diff
@@ -38,13 +35,13 @@ curl --header "Authorization: Bearer ${ADMIN_TOKEN//\"}" http://"${HOST}"/rest/d
 echo "Getting REST schema as Customer"
 
 # Create a customer
-curl --request POST --data @bin/customer.json --header 'Content-Type: application/json' http://"${HOST}"/rest/default/V1/customers
+curl --request POST --data @bin/customer.json --header 'Content-Type: application/json' "$STORE_URL"/rest/default/V1/customers
```

Comment:
Good point. I was thinking about this too. I don't want to store customer data in different places since it's already defined in `bin/customer.json`, so I'll try to reuse the data from the instead of hardcoding it.

Reply:
i'm getting an error that points to the customer token:
`bin/functions.sh: line 24: jq: command not found`

Reply:
You just need to install it:

```
brew install jq
```

Reply:
I added a Prerequisites section to the Readme

Reply:
I don't see the updated readme, but the code and everything works well!

Reply:
I forgot to push it. Now it's there.